_CodingChallenges/124-flocking-boids.md

```diff
@@ -74,6 +74,8 @@ contributions:
   - title: "Flocking in JavaFX"
     author:
       name: "Paul Keller"
+      url: "https://github.com/scorixear/"
+    url: "https://youtu.be/9B_12AXKfR4"
```

Comment:
```suggestion
    video_id: "9B_12AXKfR4"
```

Reply:
You can change it if you want. Otherwise let us know, then we will go merge.

Reply:
no that works :) Thanks